src/AddressRegistry.Api.BackOffice.Handlers.Lambda/Handlers/SqsLambdaHandler.cs

```diff
@@ -1,116 +1,41 @@
-
         protected SqsLambdaHandler(
             IConfiguration configuration,
             ICustomRetryPolicy retryPolicy,
             IStreetNames streetNames,
             ITicketing ticketing,
             IIdempotentCommandHandler idempotentCommandHandler)
+            : base(configuration, retryPolicy, ticketing, idempotentCommandHandler)
         {
-            _retryPolicy = retryPolicy;
             _streetNames = streetNames;
-            _ticketing = ticketing;
-            IdempotentCommandHandler = idempotentCommandHandler;
-
-            DetailUrlFormat = configuration["DetailUrl"];
-            if (string.IsNullOrEmpty(DetailUrlFormat))
-            {
-                throw new ConfigurationErrorsException("'DetailUrl' cannot be found in the configuration");
-            }
         }
 
-        protected abstract Task<ETagResponse> InnerHandle(TSqsLambdaRequest request, CancellationToken cancellationToken);
-
-        protected abstract TicketError? MapDomainException(DomainException exception, TSqsLambdaRequest request);
-
-        public async Task<Unit> Handle(TSqsLambdaRequest request, CancellationToken cancellationToken)
-        {
-            try
-            {
-                await ValidateIfMatchHeaderValue(request, cancellationToken);
-
-                await _ticketing.Pending(request.TicketId, cancellationToken);
-
-                ETagResponse? etag = null;
-
-                await _retryPolicy.Retry(async () => etag = await InnerHandle(request, cancellationToken));
-
-                await _ticketing.Complete(
-                    request.TicketId,
-                    new TicketResult(etag),
-                    cancellationToken);
-            }
-            catch (AggregateIdIsNotFoundException)
-            {
-                await _ticketing.Error(request.TicketId,
-                    new TicketError(ValidationErrors.Common.AddressNotFound.Message, "404"),
-                    cancellationToken);
-            }
-            catch (IfMatchHeaderValueMismatchException)
-            {
-                await _ticketing.Error(
-                    request.TicketId,
-                    new TicketError("Als de If-Match header niet overeenkomt met de laatste ETag.", "PreconditionFailed"),
-                    cancellationToken);
-            }
-            catch (DomainException exception)
-            {
-                var ticketError = exception switch
-                {
-                    AddressIsNotFoundException => new TicketError(
-                        ValidationErrors.Common.AddressNotFound.Message,
-                        ValidationErrors.Common.AddressNotFound.Code),
-                    AddressIsRemovedException => new TicketError(
-                        ValidationErrors.Common.AddressRemoved.Message,
-                        ValidationErrors.Common.AddressRemoved.Code),
-                    _ => MapDomainException(exception, request)
-                };
```

Comment:
This is not caught any more. Catch in `MapException` in this class here and use an `InnerMapDomain` ?